python/pyspark/ml/image.py

```diff
@@ -55,7 +72,7 @@ def imageSchema(self):
         """
 
         if self._imageSchema is None:
-            ctx = SparkContext._active_spark_context
+            ctx = SparkContext.getOrCreate()
```

Comment:
Have you check every place to use `getOrCreate` ?

Reply:
Yes, everywhere in image schema code at least.